tests/test_safeFileIo.py

```diff
@@ -147,14 +147,14 @@ def testNoChange(self):
             self.assertEqual(f.read(), "some test string")
 
 
-class TestOneThousandWriters(unittest.TestCase):
+class TestMultipleWriters(unittest.TestCase):
     """Test for efficient file updating with shared & exclusive locks by serializing a RepositoryCfg to a
     location 1000 times. When this was tested on a 2.8 GHz Intel Core i7 macbook pro it took about 1.3 seconds
     to run. When the squash performance monitoring framework is done, this test could be monitored in that
     system."""
 
     def setUp(self):
-        self.testDir = tempfile.mkdtemp(dir=ROOT, prefix='TestOneThousandWriters-')
+        self.testDir = tempfile.mkdtemp(dir=ROOT, prefix='TestMultipleWriters-')
```

Comment:
Why not do `self.__class__.__name__` for the prefix here, to keep it generic?

Reply:
Seems a bit too fancy for something that is scheduled to die before the end of the year.